src/integrationtests/java/com/aws/greengrass/integrationtests/status/EventFleetStatusServiceTest.java

```diff
@@ -126,9 +126,8 @@ class EventFleetStatusServiceTest extends BaseITCase {
     private ThingGroupHelper thingGroupHelper;
 
     private AtomicReference<List<FleetStatusDetails>> fleetStatusDetailsList;
-
-    private final CountDownLatch mainFinished = new CountDownLatch(1);
-
+    //during kernel launch, main component enters finished once then shutdown and finish again
```

Comment:
what is the reason for this?

Reply:
I want to make sure the main component reaches to FINISHED state twice